source/CardGames.py

```diff
@@ -138,4 +145,28 @@ def dealCards(self, numCards: int, players: PlayerList, deck: Deck):
     for player in players:
       for _ in range(numCards):
         player.addCard(deck.getCard())
-    return True
\ No newline at end of file
+    return True
+    print(player)
+
+  
+
+
+  def printAllPlayerCards(self):
+    TotalPlayers = [player1, player2, player3]
+    for i in TotalPlayers:
+      self.printPlayerCards(i)
+
+deck1 = Deck()
+player1 = Player("john")
+player2 = Player("jeff")
+player3 = Player("george")
+dealer1 = Dealer()
+
+
+dealer1.dealCards(2, [player1], deck1)
+dealer1.dealCards(3, [player2], deck1)
+dealer1.dealCards(4, [player3], deck1)
```

Comment:
I understand the multiple lines if you are giving each player a different number of cards, but if this is for testing purposes, you could also make a list of the players and give them an equal number of cards to condense this to one line